//package exception;
//
//public class TestDemo {
//     public static int func(){
//         try {
//             int ret = 10/3;
//             return ret;
//         }catch (ArithmeticException e){
//             e.printStackTrace();
//         }finally {
//             {
//                 return 10;//建议不要在finally中写return；
//             }
//         }
//     }
//     public static int
//}
